src/overlays/actors/ovl_En_Mt_tag/z_en_mt_tag.c

```diff
@@ -153,25 +153,26 @@ s32 EnMttag_GetCurrentCheckpoint(Actor* actor, PlayState* play, s32* upcomingChe
     checkpointIterator = sStartingCheckpointPerSceneExitIndex[sceneExitIndex];
 
     // Iterates through all possible checkpoints that are associated with this sceneExitIndex.
+    //! @bug - sStartingCheckpointPerSceneExitIndex is indexed out of bounds when sceneExitIndex is 18
```

Comment:
I think it would be more clear to specify that the OoB is produced because of the `checkpointIterator + 1` accesses

Reply:
There are 2 out of bounds accesses, first sStartingCheckpointPerSceneExitIndex[sceneExitIndex + 1] which causes sCheckpointPositions[checkpointIterator + 1] (and possibly sCheckpointPositions[checkpointIterator - 1]), just wasn't sure the best way to phrase it.

How would this be?
```
    //! @bug - sStartingCheckpointPerSceneExitIndex is indexed out of bounds when sceneExitIndex is 18, causing the loops upper bound to exceed what is expected.
    //         In turn, out of bounds accesses of sCheckpointPositions will occur as checkpointIterator increases past 45.
```

Reply:
Maybe keep it more simple and only talk about `sStartingCheckpointPerSceneExitIndex`, since that's the bigger offender here
```
    //! @bug - sStartingCheckpointPerSceneExitIndex is indexed out of bounds when sceneExitIndex is 18, due to the `sceneExitIndex+1` access.
```